### PR TITLE
Migrate to github actions

### LINKED ISSUE #4: Move to Github Actions for CI
One less external dependency.

### PR DESCRIPTION
**Pull request by nitinstp23**
fixes #4

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install OTP and Elixir
+        uses: actions/setup-elixir@v1.2.0
+        with:
+          otp-version: 22.1
+          elixir-version: 1.6.6
+      - name: Install dependencies
+        run: |
+          mix deps.get
```
Comment (vasuadari) [inline .github/workflows/main.yml]: can you pass option `--only test`?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Flume
 
-[![CircleCI](https://circleci.com/gh/scripbox/flume.svg?style=svg)](https://circleci.com/gh/scripbox/flume)
+![CI](https://github.com/scripbox/flume/workflows/CI/badge.svg?branch=master&event=push)
```
Comment (vasuadari) [inline README.md]: Rename `CI` to `Test`.

Comment (nitinstp23) [reply README.md]: Done!

File: .github/workflows/main.yml (hunk 1 above)
Comment (nitinstp23) [reply .github/workflows/main.yml]: Done!

**Pull request merged**